tests/vm_util_test.py

```diff
@@ -17,10 +17,56 @@
 import unittest
 
 import mock
+import os
 
 from perfkitbenchmarker import vm_util
 
 
+if os.name is 'posix':
```

Comment:
I think I'm just going to drop the tests, even if I test for making sure subprocess.Popen gets the right argument, I wouldn't be testing any new logic, it will just be a "variable assignment".